docs/tutorials/single-dev-workflow.md

```diff
@@ -0,0 +1,206 @@
+
+Let's see first how to deploy a simple application:
+
+```shell
+# Example code: https://github.com/epinio/example-12factor
+
+# Move to the source code directory. Here is an example:
+cd /github/example-12factor
+
+# Deploy your application
+epinio push -n mysimpleapp
+```
+
+At the end of the deployment output, you have the URL to be used for checking your application:
+
+```shell
+Deploying application ...
+..............
+🕞  Creating application resources
+✔️  App is online.
+Name: mysimpleapp
+Namespace: workspace
+Builder Image:
+Routes:
+1: https://mysimpleapp.127.0.0.1.sslip.io
+```
+
+### List the applications deployed
+
+If you're working on many applications, it can be really useful to see when was the last time they were deployed and which URL you should use to check them.
+
+You can get the application's information with the following two commands:
+
+```shell
+# List all deployed applications
+epinio app list
+
+# Display the applications information
+epinio app show mysimpleapp
+```
+
+### View installation logs
+
+If your application couldn't be installed, you might want to check your installation logs or, even better, save them into a file for a better screening with text editor.
+
+You can access the logs by running the command:
+
+```shell
+# Get the installation logs
+epinio app logs --staging mysimpleapp
+
+# Get the application logs
+epinio app logs --follow mysimpleapp
+```
+
+### View application logs
+
+Another type of logs that you can access is the application logs. And specially with web applications, you might want to have realtime logs displaying so you spot bugs faster.
+
+Epinio can display the logs both statically or dynamically as follows:
+
+```shell
+# Display logs statically
+epinio app logs mysimpleapp
+
+# Display logs dynamically
+epinio app logs --follow mysimpleapp
+```
+
+### Create a new port-forward
+
+As described above, Epinio creates a new `ingress route` for your application. The route is bound by default to the port `443`.
+
+However, you might need to test parts of your application using a different port. For these specific cases, you can run the following command:
+
+```shell
+epinio app port-forward 8080:8080 mysimpleapp
+```
+
+:::tip
+
+You can specify only one port number. In that case, Epinio will open the port of both `local` and `remote` targets.
+
+For more information, you can see the [Port Fowarding page](../howtos/port_forwarding.md).
+
+:::
+
+### Scale your application
+
+Another common task with Cloud Native applications, is to add (and remove) several instances of your application. This feature, called scaling, can be achieved with Epinio with the following command:
+
+```shell
+epinio app update mysimpleapp --instances 3
+```
+
+After you scaled your application up or down, you can check the status with the command:
+
+```shell
+epinio app show mysimpleapp
+```
+
+## Uninstall your application
```

Comment:
I feel like "uninstall" is the wrong word here?

Reply:
```suggestion
## Remove your application
```